src/config/lang/english.go

```diff
@@ -665,10 +665,12 @@ const (
 	AgentErrBindHandler            = "Unable to bind the webhook handler"
 	AgentErrCouldNotDeserializeReq = "could not deserialize request: %s"
 	AgentErrGetState               = "failed to load zarf state: %w"
+	AgentErrParsePod               = "failed to parse pod: %w"
 	AgentErrHostnameMatch          = "failed to complete hostname matching: %w"
 	AgentErrImageSwap              = "Unable to swap the host for (%s)"
 	AgentErrInvalidMethod          = "invalid method only POST requests are allowed"
 	AgentErrInvalidOp              = "invalid operation: %s"
+	AgentErrTransformGitURL        = "unable to transform the git url"
```

Comment:
Formatted this a bit differently (without %w) than the other errors so I could use this constant in my tests, lmk if you disagree with this approach

Reply:
although since this is only used internally, i'm fine w/ keeping it in `english`, its an easy fix later

feel free to resolve if you feel strongly enough about keeping it

Reply:
Gonna move just this one to the package for now, leaving a TODO to move the rest of the agent errors in a future refactor (there's other errors I want to move as well)